jobs/release-monitoring/release-monitoring-3scale/release-monitoring-3scale.yaml

```diff
@@ -0,0 +1,50 @@
+---
+- job:
+    name: 3scale-release-monitoring
+    display-name: 'Integreatly 3scale Release Monitoring'
+    project-type: pipeline
+    concurrent: false
+    triggers:
+      - timed: 'H */1 * * *'
+    parameters:
+      - string:
+          name: 'installationGitUrl'
+          default: 'git@github.com:integr8ly/installation.git'
+          description: '[REQUIRED] The installation repo containing the components meta file (COMPONENTS.yaml)'
+      - string:
+          name: 'installationGitRef'
+          default: 'master'
+          description: '[REQUIRED] The installation git ref'
+      - string:
+          name: 'installationProductBranch'
+          default: '3scale-next'
+          description: '[REQUIRED] The installation git branch to push new version changes'
+      - string:
+          name: 'manifestVar'
+          default: 'threescale_version'
+          description: '[REQUIRED] The manifest variable to be used as the current component version'
+      - string:
+          name: 'projectOrg'
+          default: '3scale'
+          description: '[REQUIRED] github project organization'
+      - string:
+          name: 'projectRepo'
+          default: '3scale-amp-openshift-templates'
+          description: '[REQUIRED] github project repostirory'
+      - string:
+          name: 'githubToken'
+          default: 'jenkins-github-api-token'
+          description: '[REQUIRED] github api token secret text id to be used in its rest api requests'
+      - string:
+          name: 'credentialId'
+          default: 'jenkinsgithub'
+          description: '[REQUIRED] github ssh credential id to be used'
+    pipeline-scm:
+      script-path: jobs/release-monitoring/release-monitoring-3scale/Jenkinsfile
+      scm:
+        - git:
+            branches:
+              - master
+            url: 'https://github.com/integr8ly/ci-cd.git'
+            skip-tag: true
+            wipe-workspace: false
```

Comment:
Since this is fully paramateriased and all these project release discovery jobs will be exactly the same, apart from the tag processing part, would it not make more sense to make this a generic github-release-monitoring job and create another that wraps this and passes the 3scale params into it? That's what we planned to do with the other pipelines anyway so would be more consistent and reduce the duplication.